src/components/SignUp/SignUpForm.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useCallback } from 'react';
+import Airtable from 'airtable';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import './SignUp.css';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+const INITIAL_FORM_STATE = {
+  username: '',
+  fullname: '',
+  password: '',
+};
+
+// Main fields are, username (email), password, First Name, Last Name, Role
+export default function SignUpForm() {
+  const [formState, setFormState] = useState(INITIAL_FORM_STATE);
+  const [role, setRole] = React.useState('');
+
+  const roleChange = (event) => {
+    setRole(event.target.value);
+  };
+  const onSubmit = useCallback((event) => {
+    event.preventDefault();
+    base('Users').create([
+      {
+        fields: {
+          username: formState.username,
+          password: formState.password,
+          'user type': role,
+          'display name': formState.fullname,
+          approval: 'unapproved',
+          'seller quotes': [
```

Comment:
Make sure to keep the fields like username, password, user type, etc consistent, so use single quotes for all

Reply:
Eslint throws an error for putting quotes around approval:
"Unnecessarily quoted property 'approval' found.eslint(quote-props)"

Should I figure out a way to circumvent the error?

Reply:
In that case let's refactor the names on Airtable go from user type to user-type and we'll just change that accordingly throughout our code. So change user type -> user-type, display type -> display-type, etc.